src/delays.js

```diff
@@ -13,3 +13,15 @@ export const API_DELAY = 2000;
 
 // How long the server waits for data before giving up.
 export const ABORT_DELAY = 10000;
+
+// DIRTY DIRTY way to polyfill some missing globals in miniflare
+// There is a better way: https://miniflare.dev/api.html#arbitrary-bindings
+globalThis.setImmediate = globalThis.setImmediate || ((fn) => setTimeout(fn, 0));
```

Comment:
Looks like we can use `||=` nowadays (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment)

```suggestion
globalThis.setImmediate ||= fn => setTimeout(fn, 0);
```